test/com/google/enterprise/adaptor/prebuilt/PrebuiltTransformsTest.java

```diff
@@ -69,13 +70,13 @@ public void testCopyMetadataOverwriteFalse() {
     metadata.add("author", "Fred");
     metadata.add("contributors", "Mary");
     metadata.add("contributors", "George");
-    transform.transform(metadata, new HashMap<String, String>());
+    transform.transform(metadata, new LinkedHashMap<String, String>());
```

Comment:
There are 153 lines with Map on them in this file.  In your opinion is the line-specific thinking worth it? I  did a file replace. Thanks